docs/source/installation.rst

```diff
@@ -53,62 +55,76 @@ Other DROP versions
 
 The developer version of DROP can be found in the `repository <https://github.com/gagneurlab/drop>`_ under the branch
 ``dev``.
-Make sure that the :any:`dependencies` are installed.
+Make sure that the :any:`dependencies` are installed, preferably in a conda environment.
+Then install DROP from github using ``pip``.
 
 .. code-block:: bash
 
-    # activate your python environment if you are using one, e.g. drop_env
-    conda activate drop_env
+    pip install git+https://github.com/gagneurlab/drop.git@dev
 
-Then install DROP from github using ``pip``.
-For this recursively clone the repository with all its submodules and then install from directory.
+
+Alternatively, you can clone the desired branch of the repository and install from directory.
 
 .. code-block:: bash
 
-    git clone -b dev https://github.com/gagneurlab/drop.git --recurse-submodules
+    git clone -b dev https://github.com/gagneurlab/drop.git
     pip install ./drop
 
-Alternatively, you can also install it directly without cloning
+If the package needs to be updated frequently, it is more useful to use the ``-e` option of ``pip``.
+Any new update pulled from the repository will be available without reinstall.
+Note, that this requires an explicit call to update any existing project (:any:`dropUpdate`).
 
-.. code-block:: bash
+.. code-block::
+
+    pip install -e ./drop
+
+    # update project directory
+    cd <path/to/project>
+    drop update
 
-    pip install git+https://github.com/gagneurlab/drop.git@dev
 
 .. _dependencies:
 
 Dependencies
 ------------
-The easiest way to ensure that all dependencies are installed is to install the
-`bioconda package <https://anaconda.org/bioconda/drop>`_ into a conda environment.
 
-.. code-block:: bash
-
-    conda install -c bioconda drop
-
-Other versions of drop can be installed after the bioconda package has been installed.
+The easiest way to ensure that all dependencies are installed is to install the bioconda package, as described above.
+Once the environment is set up and installation was successful, other versions of drop can be installed with ``pip``,
+overwriting the conda version of ``DROP`` (see :any:`otherversions`).
 
 
 Installation without conda
 ++++++++++++++++++++++++++
 Alternatively, DROP can be installed without ``conda``. In this case the following dependencies must be met:
```

Comment:
the commandline tools `wget` and `bc` are not always present in minimal installations. 
So we should state the dependencies here!

Also what about `wbuild` and the required R packages like `knitr` and `rmarkdown`.

Reply:
wbuild is specified in the `setup.py`, so it will be installed together with drop. All the R packages will be installed on first invocation of the pipeline and can optionally be installed beforehand, as described.
I'd say `Dependencies` is a misnomer and should be called `Prerequisites` instead.

Reply:
@c-mertes I updated the prerequisite list